.github/workflows/action.yml

```diff
@@ -18,6 +18,8 @@ jobs:
     name: Version with commit unique id
     runs-on: ubuntu-latest
     if: github.ref == 'refs/heads/main'
+    permissions:
+      contents: write
```

Comment:
This was GitHub. If you prefer I can push in a separate pull request to separate concerns.